tests/queries/0_stateless/01155_rename_move_materialized_view.sql

```diff
@@ -27,13 +31,14 @@ DROP DATABASE test_01155_ordinary;
 USE default;
 
 INSERT INTO test_01155_atomic.src VALUES ('after moving tables');
-SELECT 2, substr(_table, 1, 10), s FROM merge('test_01155_atomic', '') ORDER BY _table, s; -- { serverError 81 }
+SELECT materialize(2), substr(_table, 1, 10), s FROM merge('test_01155_atomic', '') ORDER BY _table, s; -- { serverError 81 }
```

Comment:
How materialization is connected to the issue of this PR?

Reply:
Test has found another bug (`Cannot convert column `1` because it is non constant in source stream but must be constant in result`), `materialize()` is a workaround